conf/clusterrunner-master

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
```

Comment:
This file is from a sysv template project on github. I only modified the defines at the top.

Reply:
this is cool. Can we have a similar init.d script for slaves too?

Reply:
Right. I was planning to add in a separate PR later. It's basically a copy of the current script so wasn't going to add much value to this one. I'll note it in the jira issue I have for deploying the slaves.